### PR TITLE
Reduce the size of the logos

### LINKED ISSUE #7200: [Docs] Reduce size of the logo in pipelining-service-mesh-specifications.md
It'd be nice to reduce the size of the logos (the images other than the screenshots).

_Originally posted by @leecalcote in https://github.com/meshery/meshery/issues/7110#issuecomment-1436325341_
            

---

#### Contributor [Guides](https://docs.meshery.io/project/contributing) and [Handbook](https://layer5.io/community/handbook)
- 📚 [Instructions for contributing to documentation](https://docs.meshery.io/project/contributing/contributing-docs)
   - Meshery documentation [site](https://docs.meshery.io/) and [source](https://github.com/meshery/meshery/tree/master/docs)
- 🛠 [Meshery Build & Release Strategy](https://docs.meshery.io/project/contributing/build-and-release)
- 🎨 Wireframes and designs for Meshery UI in [Figma](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI)
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by mohit-marathe**
**Notes for Reviewers**

This PR fixes #7200 
Screenshot_20230226_132644

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit f19acede555d5ac5e7f7b2c3a9667ee1ee1fb150 at: https://63fb22e8fb16ae4f1936e24b--meshery-docs-preview.netlify.app

File: docs/pages/guides/pipelining-service-mesh-specifications.md [hunk 1]
```diff
@@ -9,11 +9,11 @@ language: en
 
 With growing adoption of service meshes in cloud native environments, service mesh abstractions - service mesh-neutral specifications - have emerged. <a href="https://layer5.io/projects/service-mesh-performance">Service Mesh Performance</a>  and <a href="https://layer5.io/projects/service-mesh-interface-conformance">Service Mesh Interface</a> are two open specifications that address the need for universal interfaces for interacting with and managing any type of service mesh. Let’s examine what each specification provides.
 
-<img src="{{ site.baseurl }}/assets/img/smp-dark-text-side.svg" className="image-left-no-shadow" alt="service mesh performance logo" style="width: 50%; height: 50%;" />
+<img src="{{ site.baseurl }}/assets/img/smp-dark-text-side.svg" className="image-left-no-shadow" alt="service mesh performance logo" style="width: 35%; height: 35%;" />
```
Comment (GaganpreetKaurKalsi) [inline docs/pages/guides/pipelining-service-mesh-specifications.md:12]:
```suggestion
<img src="{{ site.baseurl }}/assets/img/smp-dark-text-side.svg" className="image-left-no-shadow" alt="service mesh performance logo" style="width: 60%; max-width: 250px" />
```

**Review by GaganpreetKaurKalsi — COMMENTED** [review]
The images spanning 35% width looks too large on large screen devices. Giving a max-width should be good. Also for small screen devices, 35% width seems too small. 
I have proposed values for both properties which seems good to me.

File: docs/pages/guides/pipelining-service-mesh-specifications.md (hunk 1 above)
Comment (mohit-marathe) [reply docs/pages/guides/pipelining-service-mesh-specifications.md]: @GaganpreetKaurKalsi, I have implemented the suggested changes.

**Comment by l5io** [comment]
🚀 Preview for commit d3464afda44094f402c4b9c08e8aa07076645a82 at: https://63fb44e70e38c163549d71a2--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit b1cd6404d6deb8bc12f30f57406d30775aa45aa8 at: https://63fc15f8e2abea6c0d935dce--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit f19acede555d5ac5e7f7b2c3a9667ee1ee1fb150 at: https://63fc17d72f15a47720377224--meshery-docs-preview.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit cee98d6802921c9f10ac38a61d92f3bd38e3bf21 at: https://63fc1bbbfb16ae7afc36e1d7--meshery-docs-preview.netlify.app

**Comment by mohit-marathe** [comment]
@leecalcote, can you please review it?

**Comment by leecalcote** [comment]
Last piece of feedback from me: please hyperlink the screenshot.

**Review by leecalcote — CHANGES_REQUESTED** [review]
Pending assignment of css class to have the screenshot open in a modal.

**Comment by l5io** [comment]
🚀 Preview for commit 2e9cdf82a4014bd1f7ceadb897fd5bf789479880 at: https://6400300593526a35037d3101--meshery-docs-preview.netlify.app

**Comment by leecalcote** [comment]
@mohit-marathe please try using this class: `lightbox-image current`.

**Comment by leecalcote** [comment]
Circling back to include in today's release. No change here.

**Comment by mohit-marathe** [comment]
> Circling back to include in today's release. No change here.

Is it ready to be merged now? I tried using `lightbox-image`, but it didn't work as it was supposed to.

**Comment by leecalcote** [comment]
@mohit-marathe will you please review the lightbox javascript and document its uses in the docs contributing page?

**Comment by leecalcote** [comment]
@mohit-marathe don't use absolute reference to images / pages within the docs site. use `{{ site.baseurl }}` with relative reference from there, please.

**Comment by leecalcote** [comment]
Please use `<Link>` for site internal URLs.

File: docs/pages/guides/pipelining-service-mesh-specifications.md [hunk 2]
```diff
@@ -9,11 +9,11 @@ language: en
 
 With growing adoption of service meshes in cloud native environments, service mesh abstractions - service mesh-neutral specifications - have emerged. <a href="https://layer5.io/projects/service-mesh-performance">Service Mesh Performance</a>  and <a href="https://layer5.io/projects/service-mesh-interface-conformance">Service Mesh Interface</a> are two open specifications that address the need for universal interfaces for interacting with and managing any type of service mesh. Let’s examine what each specification provides.
 
-<img src="{{ site.baseurl }}/assets/img/smp-dark-text-side.svg" className="image-left-no-shadow" alt="service mesh performance logo" style="width: 50%; height: 50%;" />
+<img src="{{ site.baseurl }}/assets/img/smp-dark-text-side.svg" className="image-left-no-shadow" alt="service mesh performance logo" style="width: 60; max-width: 250px;" />
```
Comment (GaganpreetKaurKalsi) [inline docs/pages/guides/pipelining-service-mesh-specifications.md:12]:
```suggestion
<img src="{{ site.baseurl }}/assets/img/smp-dark-text-side.svg" className="image-left-no-shadow" alt="service mesh performance logo" style="width: 60%; max-width: 250px;" />
```

Comment (GaganpreetKaurKalsi) [reply docs/pages/guides/pipelining-service-mesh-specifications.md:12]: Missing percentage sign.

**Comment by l5io** [comment]
🚀 Preview for commit e47537ee73e0329776ca6d2d9cfd89ef637f9558 at: https://6404191129b7613cdf3775f9--meshery-docs-preview.netlify.app

**Comment by mohit-marathe** [comment]
> @mohit-marathe will you please review the lightbox javascript and document its uses in the docs contributing page?

@leecalcote, I reviewed the `docs/assets/js/lightbox.js` file and I observed that `lightbox` will be used by default unless we add a class named `no-lightbox`. So in my opinion there is no need to document its uses as it is applied by default.

**Comment by Chadha93** [comment]
@mohit-marathe Let's discuss this on the Meshery Dev call. Please add this as an agenda item in the meeting minutes if you would. :)

**Pull request merged**